### DISCUSSION TITLE
What is known as the "♂️👑 of herbs"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#873] [Galaxy Trivia] [answered]
**Bonus:** What is known as the "♀️👑 of herbs"?

**Comment by peterbaccaro** [comment, marked as answer]
**Basil** is commonly referred to as the "king of herbs" due to its widespread use and popularity in many cuisines around the world, particularly in Mediterranean and Italian cuisine. It has a distinct aroma and flavour that adds freshness and depth to dishes, making it a favourite herb for many chefs and home cooks.

**Rosemary** is often called the "queen of herbs" because of its strong and fragrant nature. It is a versatile herb known for its woody aroma and pine-like flavour. Rosemary is commonly used in Mediterranean cuisine, especially in dishes like roasted meats, vegetables, and bread. It also has a long history of medicinal and therapeutic uses.
